config-vm.md

```diff
@@ -0,0 +1,37 @@
+# Virtual-machine-specific Configuration
+
+Virtual-machine-based runtimes require additional configuration to that specified in the [default spec configuration](config.md).
+
+This optional configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) host path to file that represents the root filesystem for the virtual machine.
```

Comment:
Hi @stevvooe - what members were you thinking would be in this structure initially aside from `path`? We could introduce an `imageType` possibly, but then the runtime could detect the type itself potentially from the file header. There isn't a `kernelType` member currently although arguably there could be, but the hypervisor we use detects the kernel type at runtime.

Also, if type members get added, presumably the behaviour would need to be documented should the type specified not match that detected by the runtime. But that implies the runtime would need to validate the type itself which makes me wonder if we anything beyond `imagePath`. Maybe such type members could be optional and default to `auto-detect` meaning the runtime is responsible for determining if it is a supported type?